doc/api/v1.yaml

```diff
@@ -46,8 +46,19 @@ paths:
                   '68dcc12fe0dfe5e7b66ca6f8c959f9aa43b273e120a77fc3e4e2f04f1ecd7968',
                 ]
         '400':
-          description: One of messages is malformed
-
+          description: An error occurred when processing this request
+          content:
+            application/json:
+              schema:
+                description: A list of fragment IDs (same order as in input)
+                type: array
+                items:
+                  type: string
+                  pattern: '[0-9a-f]+'
```

Comment:
Did not discuss this in the master PR: If we fail to parse any fragments, there won't be anything to put in this array, right?

Reply:
Yes. The master PR has a separate array for rejects though.

Reply:
Yeah, but in case of parsing error there won't be an ID even for rejects.